apps/selvbetjening-ui/src/inntektsjustering/2-inntekt-til-neste-aar/skjemaer/AttenTilFemtiSeksAarSkjema.tsx

```diff
@@ -32,7 +32,7 @@ export const AttenTilFemtiSeksAarSkjema = () => {
         defaultValues: inntekt,
     })
 
-    if (innholdError && !innholdIsLoading) {
+    if ((innholdError && !innholdIsLoading) || !innhold?.inntektSkjemaer?.attenTilFemtiSeksAar) {
         return <Navigate to="/system-utilgjengelig" />
     }
```

Comment:
tror denne virker, bør testes, gjelder andre komponenter også
```suggestion
    if ((!innholdIsLoading && (!innhold?.inntektSkjemaer?.attenTilFemtiSeksAar || innholdError)) {
        return <Navigate to="/system-utilgjengelig" />
    }
```